### ISSUE TITLE
Semicolon and emails with not tld in email addresses in version 2.0.1

### ISSUE THREAD

**Issue by elle** [K-and-R/email_validator#60]
In version `2.0.1`: `foo@example` comes back valid
In versions `1.6.0` and `2.0.1`: `foo;@example.com` comes back valid, but is invalid in version `1.4.0`

Example from my specs:
```
User validations should not allow :email to be ‹"foo"›, ‹"foo@"›, ‹"foo@example"›, ‹"foo@example..com"›, or ‹"example.com"›
     Failure/Error:
       is_expected.not_to allow_values(
         "foo",
         "foo@",
         "foo@example",
         "foo@example..com",
         "example.com",
       ).for(:email)
     
       After setting :email to ‹"foo@example"›, the matcher expected the User
       to be invalid, but it was valid instead.
```

**Comment by karlwilbur** [comment]
Hi, @elle. You are correct. There are strings that pass validation in `2.x` which would fail in `1.x`. This is expected behavior. Please read the  [Validation philisophy section of the README](https://github.com/balexand/email_validator/blob/756f4226d254713333f83f534b88d174a106eb37/README.md#validation-philosophy); this gem ([as of 2019-03-02](https://github.com/balexand/email_validator/commit/faa407b87840bb66a426c7b597610fe3681f801e)) uses an extremely loose interpretation email addresses. 

You can read a little more info in [my comment](https://github.com/balexand/email_validator/issues/58#issuecomment-489829426) on issue #58.

**Comment by karlwilbur** [comment]
This has been fixed in `2.1.0`.

I am closing this issue.
